doc/src/bond_comm_roadmap.md

```diff
@@ -10,9 +10,12 @@ If there's a feature you think is useful and should be on the roadmap or
 would like to help implement a feature, please see the
 [guidelines for contributing](https://github.com/Microsoft/bond/blob/master/CONTRIBUTING.md).
 
+## Shipped
```

Comment:
I would skip this section. It's just going to become an abbreviated reiteration of some of the changelog. :)